### PR TITLE
Docs: Improve command sample logic, rework home page with UX findings

### PR DESCRIPTION
**Pull request by akcano**

### CODE REVIEW

File: docs/explanation/essentials/moving-projects.rst [hunk 1]
```diff
@@ -105,6 +117,7 @@ before deleting the project directory:
 
 .. code:: console
 
+   $ cd /home/user/project/
```
Comment (dmitry-lyfar) [inline docs/explanation/essentials/moving-projects.rst]: One can do `workshop --project=<dir>` remove golang. I've seen cding into the dir in a few examples in our docs and was curious whether we should use the project option instead. Esp. in this case.

File: docs/tutorial/index.rst [hunk 2]
```diff
@@ -417,9 +406,7 @@ You can run an interactive shell as well:
 .. code:: console
 
    $ workshop exec golang bash
-   $ uname -a
-
-       Linux golang-bf3a1040 6.2.0-35-generic #35~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Oct  6 10:23:26 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
+   workshop@golang-cd03e2cd:/project$ uname -a
```
Comment (dmitry-lyfar) [inline docs/tutorial/index.rst:409]: hmm, I should definitely work on the default promt which is the scary instance name now. But that's a side note, I'll put it on todo.

**Review by dmitry-lyfar — APPROVED** [review]
I think this one is good, won't merge as I wanted your comment on the 'cd' bit.

File: docs/explanation/essentials/moving-projects.rst (hunk 1 above)
Comment (akcano) [reply docs/explanation/essentials/moving-projects.rst]: In brief, this is a legitimate consideration. so I'll proceed to implement it; also, at this point, the explanation also needs a testing script.

File: docs/tutorial/index.rst (hunk 2 above)
Comment (akcano) [reply docs/tutorial/index.rst:409]: Actually, it's not that bad (all parts make sense and add information, but we simply omit it at the moment), but making it carry some added value won't hurt.

**Review by dmitry-lyfar — APPROVED** [review]

**Pull request merged**